app/javascript/schools/coaches/components/school_index/SA_Coaches_CoachEditor.res

```diff
@@ -246,9 +246,10 @@ let make = (~coach, ~closeFormCB, ~updateCoachCB, ~authenticityToken) => {
     <div className="drawer-right">
       <div className="drawer-right__close absolute">
         <button
+          ariaLabel="close"
           title="close"
           onClick={_e => closeFormCB()}
-          className="flex items-center justify-center bg-white text-gray-600 font-bold py-3 px-5 rounded-l-full rounded-r-none hover:text-gray-700 focus:outline-none mt-4">
+          className="flex items-center justify-center bg-white text-gray-600 font-bold py-3 px-5 rounded-l-full rounded-r-none hover:text-gray-700 focus:outline-none focus:text-primary-500 mt-4">
           <i className="fas fa-times text-xl" />
         </button>
       </div>
```

Comment:
@irajsuhail 

- [ ] `autoFocus=true` for the first input text field in every drawer.
- [ ] Currently, input focus styling uses `focus:border-gray-500`. Can we try using `focus:ring-indigo-[value]`?